aspnetcore/tutorials/first-mvc-app/working-with-sql.md

```diff
@@ -90,35 +99,37 @@ Replace the contents of *Program.cs* with the following code:
 
 [!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Program.cs)]
 
-Test the app
+Test the app.
 
 # [Visual Studio](#tab/visual-studio)
 
-* Delete all the records in the DB. You can do this with the delete links in the browser or from SSOX.
-* Force the app to initialize (call the methods in the `Startup` class) so the seed method runs. To force initialization, IIS Express must be stopped and restarted. You can do this with any of the following approaches:
+Delete all the records in the database by deleting links in the browser, or from SSOX.
 
-  * Right click the IIS Express system tray icon in the notification area and tap **Exit** or **Stop Site**
+Force the app to initialize, calling the methods in the `Startup` class, so the seed method runs. To force initialization, IIS Express must be stopped and restarted. Stop and restart IIS Express with any of the following approaches:
```

Comment:
```suggestion
Force the app to initialize so that `Program.Main` is called. `Main` calls `SeedData.Initialize`, which initializes the database with data. To force initialization, IIS Express must be stopped and restarted. Stop and restart IIS Express with any of the following approaches:
```